jspoon/src/main/java/pl/droidsonroids/jspoon/annotation/Nullable.java

```diff
@@ -0,0 +1,11 @@
+package pl.droidsonroids.jspoon.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.FIELD })
+public @interface Nullable {
```

Comment:
I think `Nullable` is not a good name, because it is already commonly used.
I would suggest something like `@NullableField`

Reply:
If we were to change its name I would advocate for `@Skippable` , meaning - "In case of failed coversion this property will be **skipped** and mapping will continue without raising a converter exception". And that's why there is an opened issue #29 (and for all other features) to have these conversations before PR implementation not after :)

But I  would disagree if only reason for change is because its commonly used. There is `javax.inject.Inject` (jsr330) and `com.google.inject.Inject` (Guice) also `javax.inject.Named` (jsr330) and `com.google.inject.name.Named` (Guice) same with `org.springframework.context.annotation.Scope` (Spring) and `javax.inject.Scope` and the same with `@Singleton` etc.
As long as the name accurately reflects class's function you don't have to change it just because another library is using the same name. And `@Nullable` isn't even that often, only in jsr305.

Reply:
👍 for `@Skippable`

Reply:
@koral--  or maybe even even more explicit like `@SkipIfConversionError` ?
Thinking ahead - some future annotation could skip empty values for collections with some `@SkipEmptyItems`. So would be easier to understand which `@Skip*` does what. What do you think?